README.md

```diff
@@ -31,7 +31,7 @@ const ptyProcess = pty.spawn(shell, [], {
   env: process.env
 });
 
-ptyProcess.onData((data) => {
```

Comment:
Why did you change this? `on` was deprecated some time ago and I don't think it's in the .d.ts anymore

Reply:
Oh, I made that change based on `ITerminal` from `src/interfaces.ts`. Reverted it back